Tasks/UseNodeV1/make.json

```diff
@@ -12,7 +12,11 @@
     {
       "items": [
         "node_modules/azure-pipelines-tasks-utility-common/node_modules/azure-pipelines-task-lib",
-        "node_modules/azure-pipelines-tool-lib/node_modules/azure-pipelines-task-lib"
+        "node_modules/azure-pipelines-tasks-utility-common/node_modules/azure-pipelines-tool-lib",
```

Comment:
seems the same path as ^

Reply:
forget it, just was confused by UI view 😄 
![image](https://github.com/microsoft/azure-pipelines-tasks/assets/52399739/ba10f0f3-0d61-46be-895f-f6e40fe7bb57)